src/components/Label/Label.jsx

```diff
@@ -3,48 +3,52 @@ import React, { type Node } from 'react';
 import cx from 'classnames';
 import { getDataAttrs } from '../../dataUtils';
 import type { Data } from '../../types';
-import style from './style.scss';
-
-const types = {
-  PRIMARY: 'primary',
-  SECONDARY: 'secondary',
-  DISABLED: 'disabled',
-};
+import styles from './style.scss';
 
 type Props = {|
-  text: string,
-  type: string,
   children?: Node,
-  data?: Data,
   className?: string,
+  data?: Data,
+  disabled?: boolean,
+  htmlFor?: string,
```

Comment:
Why do we need `htmlFor` prop? It is not mentioned in the plasma design documentation.

Reply:
`htmlFor` is a `<label>`-specific attribute ( that is assigned the `id` of a related input tag.
When a label has this attribute (which translates to `<label for="...">`), clicking it sets focus to the relevant input, and also ties them together semantically.

Check out: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Attributes